src/features/parameters/pcc-min/pcc-min-parameters-form.tsx

```
@@ -38,64 +38,30 @@ export const styles = {
-                                <ParameterLineDirectoryItemsInput
-                                    name={FILTERS}
-                                    equipmentTypes={[EquipmentType.VOLTAGE_LEVEL]}
-                                    elementType={ElementType.FILTER}
-                                    label="pccMinParamFilter"
-                                    hideErrorMessage={false}
-                                    allowMultiSelect={false}
-                                />
-                            </Grid>
-                        </Box>
-                    )}
+            <ParameterLayout title={'PccMin'} header={renderTitleFields?.()} actions={actions} isLoading={paramsLoading}>
```

ESLint

[error] Curly braces are unnecessary here.

(react/jsx-curly-brace-presence)